templates/file_crash_bug_description.md.jinja

```diff
@@ -31,7 +31,7 @@ Top {{ frames|length }} frames of crashing thread:
 By querying {{ signature_details_channel }} crashes reported within the last {{ signature_details_delta }}, here are some insights about the signature:
 
 - **First crash report:** {{ signature.first_crash_date }}
-- **Crash kind:** {{ "Plugin Crash" if signature.is_plugin_crash  else  "Browser Crash" }}
```

Comment:
Maybe this should be renamed something like "Process type" now instead of "Crash kind".

Reply:
See https://github.com/mozilla/bugbot/pull/2252#discussion_r1372095006

Reply:
BTW. If you think this information is not helpful to be in the bug report, we could drop it altogether.

Reply:
Well, personally I think the "browser crash" vs "plugin crash" distinction is outdated. I think it comes from the pre-e10s era where we only had two processes, one for the UI and web content, and one for Flash. Flash crashes used to be a major source of instability, but nowadays I think it only applies to EME and they appear to be pretty rare. So, personally I'd like the process type to be available and I don't care about some kind of "crash type" with coarser buckets, but Socorro does have a couple of different "Crashers" buckets so maybe somebody finds it useful. https://crash-stats.mozilla.org/home/product/Firefox

Reply:
Thank you for the context. I will change it to show the process types instead.